src/mame/mr/nl_dribling.cpp

```diff
@@ -0,0 +1,494 @@
+// license:BSD-3-Clause
+// copyright-holders: Enzo Lombardi
```

Comment:
Is there a reason for using the BSD license?  The content of a netlist is not eligible for copyright protection (it doesn’t qualify as “creative”), so the only parts that are eligible for copyright protection are things like the formatting.  It’s usually less trouble to just use CC0-1.0 for things with minimal “creative” content.

Reply:
Done